roles/ood-wrapper/defaults/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+ood_source_version: "v1.7.6"
+ood_config_dir: /etc/ood/config
+
+# Cluster configuration
+ood_cluster_config_dir: '{{ ood_config_dir }}/clusters.d'
+ood_cluster_title: "NVIDIA DeepOps Cluster"
+ood_cluster_name: "deepops"         # has to match Slurm cluster name
+ood_install_linuxhost_adapter: yes
+
+# List of extra OOD "apps" to install
+ood_install_apps:
+  bc_osc_codeserver:
+    repo: https://github.com/OSC/bc_osc_codeserver.git
+    dest: /var/www/ood/apps/sys
+    version: gtc
+
+# OOD front page config
+ood_frontpage_config : |
+  pun_custom_env:
+    OOD_DASHBOARD_TITLE: "NVIDIA DeepOps Cluster"
+    OOD_BRAND_BG_COLOR: "#76B900"
+    OOD_BRAND_LINK_ACTIVE_BG_COLOR: "#333"
+    OOD_NAVBAR_TYPE: "inverse"
+
+# Virtual Desktop app configuration
+ood_desktops_dir: /etc/ood/config/apps/bc_desktop
+ood_desktop_app_title: "NVIDIA DeepOps Desktop"
+ood_desktop_app_dir: /var/www/ood/apps/sys/bc_desktop
+
+# VS Code Server app configuration
+ood_codeserver_app_dir: /var/www/ood/apps/sys/bc_osc_codeserver
+ood_codeserver_app_def_gpus: 1
+ood_codeserver_app_cpus_per_gpu: 4
```

Comment:
Assuming that this is what is causing the failure when running in virtual?
![image](https://user-images.githubusercontent.com/3503731/77031000-ba3f7600-6976-11ea-95a9-944719bd1ff3.png)

Reply:
Was this on centos or Ubuntu?

Reply:
Ubuntu, but a virtual environment. The slurm node has 8 CPUs and 16 GB of RAM (and 4 GPUs, but 1 isn't working).